Only mark threads as read if the window is focused (#1816)
liveblocks/liveblocks: CHANGELOG.md

- Add support for custom links to the composer. (either by pasting URLs with
  plain text selected or by pasting existing links)
- Preserve whitespace and empty lines in comments.

> - Take into account the window's focus state when marking threads as read once
>   they are visible.

Reviewer: What do you think of describing it like this?

```suggestion
- Mark threads as read when visible (like before), but only if the window is focused
```
